src/views/Home.vue

```diff
@@ -1,34 +0,0 @@
-<template>
-  <Suspense>
```

Comment:
What's Suspense?

Reply:
Oh man, you're gonna love Suspense. Suspense is "a special component that renders a fallback content instead of your component until a condition is met". [You can read more about it here](https://vueschool.io/articles/vuejs-tutorials/suspense-new-feature-in-vue-3/). 

This made it really easy for me to render fallback content rather than using the `vue-if` statements with a boolean that waits for async operations to finish. 

Unfortunately, it's only supported for Vue 3, so it wouldn't really apply to y'all unless you guys choose to migrate to Vue 3 (which sounds like a p good idea imo but I imagine this would be a huge task)

Reply:
I'm loving the sound of it already. Someone on our team made a component called `StatusIndicator` which essentially does the same thing. If data is loading, it "fallback" to a loading circle. Once it's loaded, it'll show the data